Build/Customizations/Projects.props

```diff
@@ -2,9 +2,52 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
-    <TestingSetupBuildFile>$(SolutionDirectory)Build\TestingSetupForDatabase.build</TestingSetupBuildFile>
+    <TestSetupBuildFileForDatabase>$(SolutionDirectory)Build\TestSetupForDatabase.build</TestSetupBuildFileForDatabase>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <DatabaseTestConfiguration Condition="'$(DatabaseTestConfiguration)' == ''">
+      Win_NET48 + NET48 + NoBrowser + SqlServer2012 + Debug + x86;
+      Win_NET48 + NET48 + NoBrowser + SqlServer2012 + Release + x86;
+      Win_NET48 + NET48 + NoBrowser + SqlServer2012 + Debug + x64;
+      Win_NET48 + NET48 + NoBrowser + SqlServer2012 + Release + x64;
+
+      <!-- Local-->
+      LocalMachine + NET48 + NoBrowser + SqlServerDefault + Debug + x86;
+      LocalMachine + NET48 + NoBrowser + SqlServerDefault + Release + x86;
+      LocalMachine + NET48 + NoBrowser + SqlServerDefault + Debug + x64;
+      LocalMachine + NET48 + NoBrowser + SqlServerDefault + Release + x64;
+
+      <!-- Exercise compatibility between installed .NET version, target framework and SQL Server -->
+      Win_NET462 + NET462 + NoBrowser + SqlServer2014 + Release + x64;
```

Comment:
why is this first line here?

Reply:
please order the sql servers descending. my ordering was intended to push the older sql servers to the end of the list while keeping the most up to date platforms at the top of the list.

Reply:
> 
> 
> why is this first line here?

Because it is there with Win_NET472/NET472 and Win_NET48/NET472, and I have no information which configs to test with which DBs,